build.gradle

```diff
@@ -27,7 +27,14 @@ test {
         showStandardStreams = false
     }
 }
-
+task Checkstyle(type: Checkstyle) {
```

Comment:
Since we already have a checkstyle gradle task which are the `checkstyleMain` and `checkstyleTest`, is this the same as those 2?

Reply:
Yes it is, my checkstyle was not working for some reason so I had to this manually.

Reply:
Did you try re-cloning the project?

Reply:
Like deleting it and initializing it again?
I did not do that

Reply:
As in you don't have to delete the current folder you are working on, just clone your fork in a separate folder and open it with intellij. Then test the checkstyle there. If it works, you can remove this section.